dev_apps/uPnP-test/js/app.js

```diff
@@ -0,0 +1,267 @@
+(function(exports) {
+  var DEBUG = false;
+  'use strict';
+
+  var savedServices = {};
+
+  var debugEl;
+  var audioPlayer;
+  var videoPlayer;
+  var imagePlayer;
+  var discoverButton;
+  var currentPlayer;
+  var playerToggler;
+  var folderList;
+
+  function debugLog(msg, level) {
+    if (!DEBUG && level == 'debug') {
```

Comment:
We usually use `===` for stuff like this. Not a big deal, since this is a test app, but it might be worth fixing if our goal is to turn this into a real app.